rapid/lib/work_request.py

```diff
@@ -56,19 +56,23 @@ def prepare_to_send(self, data_in):
         if isinstance(data_in, dict):
             for attr in vars(self).keys():
                 try:
-                    tmp = attr
-                    if tmp == '_grain':
-                        tmp = 'grain'
-                    setattr(self, attr, data_in[tmp])
+                    if attr == '_grain':
+                        for g_attr in ['_grain', 'grain']:
+                            if g_attr in data_in:
+                                setattr(self, '_grain', data_in[g_attr])
+                    else:
+                        setattr(self, attr, data_in[attr])
```

Comment:
getting prepared for an attribute rename? `_grain` and `grain`?

Reply:
No, this is backward compatibility that wasn't working.